README.md

```diff
@@ -32,12 +32,11 @@ Currently the only status we update in Oracle is if the message has been read on
 
 ### Setup
 
-Dependencies are managed using pipenv. To install the dependencies and activate the virtual environment, run:
+Dependencies are managed using poetry. To install the dependencies and use the virtual environment, run:
 
 ```bash
-pip install pipenv
-pipenv install --dev
-pipenv shell
+pip install poetry
+poetry install --no-root --only dev
```

Comment:
Might not be understanding this but doesn't this only install dev packages ie. https://github.com/NHSDigital/bcss-notifications/pull/98/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R46 ?